features/shared/continuing_after_success.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Show clear error if trying to continue after executing a successful command
```

Comment:
Cool idea to create a "shared" feature folder. This prevents us from having to repeat such concerns that are orthogonal to the individual commands unnecessarily. Let's keep our eyes open for opportunities to dry up some other orthogonal features into this folder!